onmt/translate/process_zh.py

```diff
@@ -0,0 +1,40 @@
+from pyhanlp import HanLP
+from snownlp import SnowNLP
+import pkuseg
+
+
+# Chinese segmentation
+def ZH_segmentator(line):
+    segments = pkuseg.pkuseg().cut(line)
+    sequence = " ".join(segments)
+    return sequence
```

Comment:
We could save a few lines by returning directly `" ".join(pkuseg.pkuseg().cut(line)`. 
Same in functions below.